.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-22.04, actuated-arm64-4cpu-16gb, macos-12, windows-2019]
```

Comment:
I saw the log 

```
 systemctl restart multipathd.service packagekit.service php8.3-fpm.service polkit.service rsyslog.service runner-provisioner.service udisks2.service walinuxagent.service
```

The `runner-provisioner.service` is related to runner. It seems like that restarting it will cancel the job.

Reply:
Will test with masking the service and see if that will help

Reply:
I think its based on whether some packages are installed or not, the restart happens.  Got past the setup stage, but failed in boot here. https://github.com/akhilerm/containerd/actions/runs/9386063338/job/25845717196

Reply:
The kernel vmlinux has been updated. No idea why it restarts. 😢

Reply:
FIxed it by using vagrant builds for ubuntu 22 in ubuntu 24 runners